mypaint-brush.c

```diff
@@ -22,6 +22,18 @@
 #include <math.h>
 #include <assert.h>
 
+#define GETTEXT_PACKAGE "libmypaint"
+
+#ifdef HAVE_GETTEXT
```

Comment:
@QuLogic I just copy from [mypaint-brush-settings.c](https://github.com/mypaint/libmypaint/blob/master/mypaint-brush-settings.c#L26) 

I am not sure which is need to use GLib l18n support. it is just add the headers:

    #include <glib.h>
    #include <glib/gi18n.h>

Or it needs another modification more? Thanks for the review